### PR TITLE
Update daikin documentation (#36475)

### PR DESCRIPTION
**Pull request by fpytloun**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/36475
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch. https://github.com/home-assistant/core/pull/36475
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/daikin.markdown [hunk 1]
```diff
@@ -81,6 +82,10 @@ The `daikin` sensor platform integrates Daikin air conditioning systems into Hom
 Some models only report outside temperature when they are turned on.
 </div>
 
+<div class='note'>
+Some models does not have humidity sensor.
```
Comment (klaasnicolaas) [inline source/_integrations/daikin.markdown]: Maybe it's better to merge the three notes to one that covers all?

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_integrations/daikin.markdown (hunk 1 above)
Comment (fpytloun) [reply source/_integrations/daikin.markdown]: Maybe, I wanted to keep it separate as it's different information but if you like I'll merge it.

Comment (frenck) [reply source/_integrations/daikin.markdown]: I would suggest on a single one as well, wording suggestion:

```text
The reporting of power/energy consumption, humidity or outside temperature depends on the device model.
```

**Review by klaasnicolaas — APPROVED** [review]
✅ Approved

**Pull request merged**